lib/http.js

```diff
@@ -54,17 +54,12 @@ bedrock.events.on('bedrock-express.configure.routes', app => {
       res.json(record);
     }));
 
-  // get one or more issuer instances
+  // get issuer instances
   app.get(
     routes.instances,
     ensureAuthenticated,
     asyncHandler(async (req, res) => {
-      const {id, controller} = req.query;
```

Comment:
Can you confirm that this API was not being used in `bedrock-web-vc-issuer` to get data for a single instance?

Reply:
Yes I can confirm that it is setup properly.